corehq/apps/reports/filters/users.py

```diff
@@ -315,13 +316,18 @@ def user_es_query(cls, domain, mobile_user_and_group_slugs):
     @classmethod
     def pull_users_and_groups(cls, domain, mobile_user_and_group_slugs,
                               include_inactive=False, limit_user_ids=None):
-        user_ids = cls.selected_user_ids(mobile_user_and_group_slugs)
+        user_ids = set(cls.selected_user_ids(mobile_user_and_group_slugs))
         user_types = cls.selected_user_types(mobile_user_and_group_slugs)
         group_ids = cls.selected_group_ids(mobile_user_and_group_slugs)
+        location_ids = cls.selected_location_ids(mobile_user_and_group_slugs)
         users = []
 
+        if location_ids:
+            for location_id in location_ids:
+                user_ids |= set(get_user_ids_by_location(domain, location_id))
```

Comment:
right, that makes sense. not sure, but something that was explicit like the sister call:
```
user_ids_at_locations_and_descendants
user_ids_at_locations_and_no_descendants
```
that's kind of verbose and i'm not sure how much i love it, but it would definitely make it clear to the caller that you are not getting the entire tree